### PR TITLE
Enable automatic release on merge to master

### PR DESCRIPTION
**Pull request by Ryudo302**
See: https://www.jenkins.io/doc/developer/publishing/releasing-cd/

PR for the _repository-permissions-updater_ repo: https://github.com/jenkins-infra/repository-permissions-updater/pull/3777

```[tasklist]
### Submitter checklist
- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue
```

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 version: 2
 updates:
-- package-ecosystem: maven
+- package-ecosystem: github-actions
```
Comment (NotMyFault) [inline .github/dependabot.yml:3]: This is incorrect in terms of CD

**Review by NotMyFault — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (Ryudo302) [reply .github/dependabot.yml:3]: Hi, @NotMyFault!

I just followed the instructions from here: https://www.jenkins.io/doc/developer/publishing/releasing-cd/

image

Comment (Ryudo302) [reply .github/dependabot.yml:3]: Oh, nevermind, I think it was my misinterpretation. I should add the github-actions configuration, not replace the original with it. I will fix it.

Comment (NotMyFault) [reply .github/dependabot.yml]: I was about to say that but no worries :)

File: pom.xml [hunk 2]
```diff
@@ -9,13 +9,12 @@
...
 	<artifactId>google-chat-notification</artifactId>
-	<version>${revision}${changelist}</version>
+	<version>${changelist}</version>
 	<packaging>hpi</packaging>
 
 	<name>Google Chat Notification</name>
 	<description>Google Chat Notification for build status</description>
-	<url>
-		https://wiki.jenkins-ci.org/display/JENKINS/Google+Chat+Notification+Plugin</url>
+	<url>https://github.com/jenkinsci/google-chat-notification-plugin</url>
```
Comment (NotMyFault) [inline pom.xml:17]: Optimally this should be the url of plugins.jenkins.io linking to your plugin

**Review by NotMyFault — APPROVED** [review]

**Pull request merged**